airbyte_cdk/logger.py

```diff
@@ -27,7 +30,7 @@
     "handlers": {
         "console": {
             "class": "logging.StreamHandler",
-            "stream": "ext://sys.stdout",
```

Comment:
Should we be worried that some logs would be emitted before (maybe during `run.py`) or after (when there is a breaking exception and we catch the uncaught exception) the `with PRINT_BUFFER` statement?

Reply:
> I would also like a more exhaustive description to understand what caused the issue before and how this change fixes the issue.

@maxi297 , my guess is that issue was caused due to the logger.




> Should we be worried that some logs would be emitted before (maybe during run.py) or after (when there is a breaking exception and we catch the uncaught exception) the with PRINT_BUFFER statement?

not really, since we share exactly the same `object` (stream) in both logger.write and print statements.